script/node-disabled-tests.json

```diff
@@ -117,6 +117,7 @@
   "parallel/test-trace-events-v8",
   "parallel/test-trace-events-vm",
   "parallel/test-trace-events-worker-metadata",
+  "parallel/test-wasm-web-api",
```

Comment:
what breaks here?

Reply:
@nornagon i can add a follow-up for this but what's breaking would require a refactor far (imo) beyond the scope of this PR. Essentially, `WebAssembly.compileStreaming` and `WebAssembly.instantiateStreaming` are `undefined` when they shouldn't be. That's happening because Node.js' logic for enabling them [here](https://github.com/nodejs/node/pull/42701/files#diff-5f2fc380364ef43210af4a28c08f9a0c7ec5f85e3ec1f5c1df5e0204eb9dc721R261) is called via `node::SetIsolateUpForNode`, which by necessity can only be called on an _existing_ isolate. However, the switch to enable it [here](https://source.chromium.org/chromium/chromium/src/+/main:v8/src/wasm/wasm-js.cc;l=2983-2988?q=wasm_streaming_callback&ss=chromium%2Fchromium%2Fsrc) is called by gin during [its own isolate initialization](https://github.com/electron/electron/blob/f2c341b655406434b5a44bc8b1e2418d664c319b/shell/app/node_main.cc#L162), which means that it ends up being `undefined`. I can look into a good way to fix this but imo impact is low and refactor burden is high so i'd prefer to just disable it for now.

Reply:
Opened https://github.com/electron/electron/issues/36282

Reply:
Fair enough, happy to leave this for a followup, but it should probably be a release blocker as those methods are expected to be available in this version of Node.